### DISCUSSION TITLE
SAFE procedure phase 1, 1% rule

### DISCUSSION THREAD

**Discussion by RiannevanderFeen** [asreview/asreview#1856] [Q&A] [answered]
@Rensvandeschoot, I have a question for you:  If I was to screen at least 1% of the total dataset, and the dataset contains 160,000 records, this would mean I am to screen 1,600 records randomly. However, it seems to me totally undoable to screen 1,600 records just in de first phase alone... and 10% in phase 2 for that matter! 

In the cheat sheet you say that the reviewers can fill in the blanks when it comes to the percentages. In that case we would say 0.1 % in phase 1 (=160 records) and 1% in phase 2 (1,600 records). But then my question is: can I still state that we used the SAFE procedure? 

I realize, I am having a hard time with this level of flexibility in a procedure :-D

**Comment by FelixWdm** [comment, marked as answer]
Hi @RiannevanderFeen , I see the issue, you want to work safely and use the SAFE procedure. The SAFE Procedure is described with numbers and percentages, but it's not stated that these can't be changed. The entire process of a systematic review has a lot of variables. The aim was to give people insight in the various options there are to introduce more robust/safe steps in their AI-aided process. I would encourage you to develop your own method and share your findings here, or in a formal publication 'The SAFE-procedure in practice ...' ;-).

**Reply by RiannevanderFeen** [reply]
@FelixWdm, thank you! I actually changed one database, as two others gave records around 10,000. One database gave 160.000 hits with the same search string. That seemed a bit off. So the SAFE procedure percentage guideline also gave me good reason to pick a more relevant database :-). In that sense, a guideline can also assist in making the right choices at different levels as a guideline is not completely flexible, but off course, should not be too ridged at the same time. I will definitely look for ways to describe our methods and recommendations (in our future publication), in terms of working with ASReview lab for the purpose of our scoping review!

**Reply by Rensvandeschoot** [reply]
Working with a huge number of records is always challenging! For a recent project, we decided to use the records from the training and calibration phase as prior knowledge..

**Reply by RiannevanderFeen** [reply]
Yes, we have prior knowledge from our pilots (calibration phase) and kept it aside. We used only 1 record as prior knowledge + the 1 percentage random records to warm up the model (phase 1). (As it was recommended in the course to not use an overkill of prior knowledge).

While in phase 2, labelling all the records (up to 10%/ 2xRR_t), we look for recognizing the list of prior knowledge. In that sense, we would like to have a better search option in ASReview lab to see if we already came across all prior knowledge, as this is one of the stopping rules. 

Now I used 'ctrl f', but that doesn't seem to work properly all the time. So this would be a suggestion to upgrade the software perhaps with a search mode in the history section.

:-)

**Reply by Rensvandeschoot** [reply]
I like this suggestion! 
Could you maybe add your ideas about how this should look to this thread: https://github.com/asreview/asreview/issues/1787
Many thanks!
